ckanext/datagovcatalog/tests/test_notifications.py

```diff
@@ -85,3 +85,39 @@ def _create_harvest_source_with_owner_org_and_job_if_not_existing(self):
             )
         
         return context, harvest_source['id']
+
+    def test_create_harvest_source_with_no_org(self):
+        context, source_id = self._create_harvest_source_with_no_org()
+
+        new_rec_action = toolkit.get_action("harvest_get_notifications_recipients")
+        new_recipients = new_rec_action(context, {'source_id': source_id})
+
+        assert_in({'name': u'default', 'email': None}, new_recipients)
```

Comment:
```suggestion
        assert_in({'name': u'test.ckan.net', 'email': None}, new_recipients)
```
to fit CI environment